README.md

```diff
@@ -57,7 +57,7 @@ jobs:
       contents: write
     steps:
     - uses: actions/checkout@v2
-    - uses: ncipollo/release-action@v1
+    - uses: ncipollo/release-action@v1.11.1
```

Comment:
v1 is actually correct here. v1 always points to the latest 1.x.x version.

Reply:
Ah, I see... https://michaelheap.com/semantic-versioning-for-github-actions/